cf-key/cf-key.c

```diff
@@ -346,15 +356,17 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
             break;
 
         default:
-            {
-                Writer *w = FileWriter(stdout);
-                WriterWriteHelp(w, "cf-key", OPTIONS, HINTS, false, NULL);
-                FileWriterDetach(w);
-            }
+            PrintHelp();
             DoCleanupAndExit(EXIT_FAILURE);
 
         }
     }
 
+    if (NO_TRUNCATE && !SHOWHOSTS)
+    {
+        Log(LOG_LEVEL_ERR, "--no-truncate / -N option is only for --show-hosts / -s");
+        DoCleanupAndExit(EXIT_FAILURE);
```

Comment:
Shouldn't this also print help?

Reply:
I tried both, but I found the big help menu to be too distracting. I don't have a strong opinion.

Reply:
Added back in.